packages/building-blocks/package.json

```diff
@@ -9,6 +9,7 @@
   "module": "esm/index.js",
   "private": false,
   "dependencies": {
+    "color-contrast": "^1.0.0",
```

Comment:
There are already some methods in core-utils that do this! No need for an extra library